libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -279,6 +287,8 @@ bool AC_Fence::pre_arm_check_alt(const char* &fail_msg) const
 bool AC_Fence::pre_arm_check(const char* &fail_msg) const
 {
     fail_msg = nullptr;
+    const float alt_max_relhome = get_alt_max_relhome();
+    const float alt_min_relhome = get_alt_min_relhome();
```

Comment:
Any reason to use home as the basis for these comparisons?  Users can fly without a home set, it would be nice to not require home in here.